embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/spans/TracingApi.kt

```diff
@@ -132,6 +132,12 @@ internal interface TracingApi {
         events: List<EmbraceSpanEvent>?
     ): Boolean
 
+    /**
+     * Return the [EmbraceSpan] corresponding to the given spanId if it is active or it has completed in the current session
```

Comment:
nit: add a note under what conditions this will return null

Reply:
Ooh good call